hadoop-hdds/docs/content/beyond/Containers.zh.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Ozone 中的容器技术"
+summary: Ozone 广泛地使用容器来进行测试，本页介绍 Ozone 中容器的使用及其最佳实践。
+weight: 2
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone 的开发中大量地使用了 Docker，包括以下三种主要的应用场景：
+
+* __开发__:
+     * 我们使用 docker 来启动本地伪集群（docker 可以提供统一的环境，但是不需要创建镜像）。
+* __测试__:
+     * 我们从开发分支创建 docker 镜像，然后在 kubernetes 或其它容器编排系统上测试 ozone。
+     * 我们为每个发行版提供了 _apache/ozone_ 镜像，以方便用户体验 Ozone。
+     这些镜像 __不__ 应当在 __生产__ 中使用。
+
+<div class="alert alert-warning" role="alert">
+当在生产中使用容器方式部署 ozone 时，我们<b>强烈</b>建议你创建自己的镜像。请把所有自带的容器镜像和 k8s 资源文件当作示例指南，参考它们进行定制。
+</div>
+
+* __生产__:
+     * 我们提供了如何创建用于生产的 docker 镜像的文档。
+
+下面我们来详细地介绍一下各种应用场景：
+
+## 开发
+
+Ozone 安装包中包含了 docker-compose 的示例目录，用于方便地在本地机器启动 Ozone 集群。
+
+使用官方提供的发行包：
+
+```bash
+cd compose/ozone
+docker-compose up -d
+```
+
+本地构建方式：
+
+```bash
+cd  hadoop-ozone/dist/target/ozone-*/compose
+docker-compose up -d
+```
+
+这些 compose 环境文件是重要的工具，可以用来随时启动各种类型的 Ozone 集群。
+
+为了确保 compose 文件是最新的，我们提供了验收测试套件，套件会启动集群并检查其基本行为是否正常。
+
+验收测试也包含在发行包中，你可以在 `smoketest` 目录下找到各个测试的定义。
+
+你可以在任意 compose 目录进行测试，比如：
+
+```bash
+cd compose/ozone
+./test.sh
+```
+
+### 实现细节
+
+`compose` 测试都基于 apache/hadoop-runner 镜像，这个镜像本身并不包含任何 Ozone 的 jar 包或二进制文件，它只是提供其了启动 ozone 的辅助脚本。
```

Comment:
I think it is better to use "Ozone" for consistency in this part.
```suggestion
`compose` 测试都基于 apache/hadoop-runner 镜像，这个镜像本身并不包含任何 Ozone 的 jar 包或二进制文件，它只是提供其了启动 Ozone 的辅助脚本。
```

Reply:
Yes, this puzzled me too, the English doc sometimes use both ozone and Ozone, I am thinking maybe it has a difference in English, does it ? @elek But I believe we only need one in Chinese. @xiaoyuyao

Reply:
@iamabug We should use `Ozone` with capital `O` here as it's referred to as a product name IMO.

And [the English doc](https://github.com/apache/hadoop-ozone/blob/master/hadoop-hdds/docs/content/beyond/Containers.md#implementation-details) should be fixed.